common/entrypoint.sh

```diff
@@ -2,6 +2,12 @@
 
 set -eo pipefail
 
+if ! whoami &> /dev/null; then
+  if [ -w /etc/passwd ]; then
+    echo "siu-toba:x:$(id -u):0:SIU toba:${HOME}:/sbin/nologin" >> /etc/passwd
```

Comment:
El usuario debería ser "siu" simplemente.

Me queda la duda, con esto evitamos que se use como login esta cuenta de usuario. ¿Afecta a la administración de un contenedor? (supongamos "docker exec .. ")?

Reply:
Si, lo sé... Pero ese usuario ya existe. Esto crea un usuario solamente si se corre con un UID aleatorio. Una buena práctica propuesta desde seguridad. Lo vemos en una reunión

Reply:
entonces podemos dejar como `siu-random` o algo similar, que de a entender esa situación sucede.

Reply:
Tal vez un comentario sea mas claro. Un comentario con link al sitio de redhat que sugiere esta mejora

Reply:
El enlace es este: https://developers.redhat.com/blog/2020/10/26/adapting-docker-and-kubernetes-containers-to-run-on-red-hat-openshift-container-platform#privileged_ports